src/components/activity-groups-list/activity-group-item/activity-group-item.tsx

```diff
@@ -25,45 +31,75 @@ interface Props {
 export const ActivityGroupItem: FC<Props> = ({ group }) => {
   const styles = useActivityGroupItemStyles();
 
+  const {
+    transactionType,
+    transactionSubtype,
+    transactionHash,
+    destination: [label, value, address]
+  } = useActivityGroupInfo(group);
   const nonZeroAmounts = useNonZeroAmounts(group);
 
   const selectedRpcUrl = useSelectedRpcUrlSelector();
-  const { isTezosNode } = useNetworkInfo();
 
-  const firstActivity = group[0] ?? emptyActivity;
+  const [areDetailsVisible, setAreDetailsVisible] = useState(false);
 
   return (
-    <View style={styles.container}>
-      <Divider size={formatSize(8)} />
-      <View style={styles.upperContainer}>
-        <ActivityGroupType group={group} />
-
-        <View style={styles.exploreContainer}>
-          <PublicKeyHashText
-            style={styles.accountPkh}
-            publicKeyHash={firstActivity.hash}
-            testID={ActivityGroupItemSelectors.operationHash}
-          />
-          <Divider size={formatSize(4)} />
-          <ExternalLinkButton
-            url={tzktUrl(selectedRpcUrl, firstActivity.hash)}
-            testID={ActivityGroupItemSelectors.externalLink}
-          />
+    <View style={styles.root}>
+      <View style={[styles.row, styles.justifyBetween]}>
+        <View style={[styles.row, styles.itemsStart]}>
+          <RobotIcon size={formatSize(36)} seed={transactionHash} />
+          <Divider size={formatSize(10)} />
+          <View>
+            <Text style={styles.oprationTitle}>{transactionType}</Text>
+            <Text style={styles.oprationSubtitle}>
+              {label} {value}
+            </Text>
+          </View>
+        </View>
+        <View>
+          <ActivityGroupAmountChange nonZeroAmounts={nonZeroAmounts} />
+          <ActivityGroupDollarAmountChange nonZeroAmounts={nonZeroAmounts} />
         </View>
       </View>
-      <Divider size={formatSize(8)} />
-      <ActivityGroupAmountChange nonZeroAmounts={nonZeroAmounts} />
-      <Divider size={formatSize(4)} />
-      <View style={styles.lowerContainer}>
-        <View style={styles.statusContainer}>
-          <ActivityStatusBadge status={firstActivity.status} />
+      <Divider size={formatSize(12)} />
+      <View style={[styles.row, styles.justifyBetween]}>
+        <View style={styles.row}>
+          <ActivityStatusBadge status={group[0].status} />
           <Divider size={formatSize(4)} />
-          <ActivityTime timestamp={firstActivity.timestamp} />
+          <ActivityTime timestamp={1689343447} />
```

Comment:
const FIRST_ACTIVITY_TIMESTAMP = 1689343447;

Reply:
Looks like a mock value... Could you replace it with actual one?